docs/guides/src/Domain/Project.cs

```diff
@@ -16,6 +16,11 @@ public static Project Create(string name)
         return new Project { Id = SId<Project>.New(), Name = name, };
     }
 
+    public void AddTasks(IEnumerable<Task> tasks)
```

Comment:
Considering the fact that `Task` is identifiable and has a reference to `Project` I'd say we should not allow the `Task` creation outside the `Project` aggregate. 

If `Task` would be `ValueObject` the approach with passing it from service level is fine.